payment-instruments/mobile-pay/seamless-view.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Seamless
+redirect_from: /payments/mobile-pay/seamless-view
+estimated_read: 10
+description: |
+  **Seamless View** scenario gives your customers the opportunity to pay with
+  MobilePay directly from within your webshop. In the Seamless view scenario,
+  Swedbank Pay receives a mobile number (MSISDN) from the payer through
+  Swedbank Pay Payments. Swedbank Pay performs a payment that the payer then
+  must confirm through the MobilePay mobile app.
+menu_order: 700
+---
+## Step 1: Create a purchase
+
+When properly set up in your merchant/webshop site and the payer starts the
+purchase process, you need to make a `POST` request towards Swedbank Pay with
+your Purchase information. This will generate a payment object with a unique
+`paymentID`. An example of an abbreviated `POST` request is provided below. You
+will receive a response in which you can find the **JavaScript source** in the
+`view-payment` operation. An example of an expanded `POST` request is available
+in the [other features section][purchase].
+
+{% include alert-gdpr-disclaimer.md %}
```

Comment:
Note: I did not include the `{% include alert-risk-indicator.md %}` line, as MobilePay does not currently support the risk indicator fields.